### PR TITLE
Add health check, fix travis

### PR DESCRIPTION
**Pull request by yakutovicha**

### CODE REVIEW

**Comment by yakutovicha** [comment]
ready for the reivew

File: .travis.yml [hunk 1]
```diff
@@ -19,11 +19,12 @@ addons:
...
+    - while [[ \"healthy\" != `docker inspect --format='{{json .State.Health.Status}}' $DOCKERID` ]]; do sleep 1; done
 
 # I just run a command with verdi that returns a zero error code to see if
 # verdi is installed properly
 script:
-    - sleep 30 # wait 30 seconds till the container is ready
-    - "export DOCKERID=`docker ps -qf 'ancestor=aiida-docker-stack'`"
     - "echo \"Docker ID: $DOCKERID\""
-    - "docker exec --tty --user aiida \"$DOCKERID\" /bin/bash -l -c 'verdi status'"
+    - "docker exec --tty --user aiida \"$DOCKERID\" /bin/bash -l -c '/usr/lib/postgresql/10/bin/pg_ctl -D /home/$SYSTEM_USER/.postgresql status'" # checking postgres
```
Comment (ltalirz) [inline .travis.yml:29]: Would this also work?
If yes, then that would not break with the next postgres update
```suggestion
    - "docker exec --tty --user aiida \"$DOCKERID\" /bin/bash -l -c 'pg_ctl -D /home/$SYSTEM_USER/.postgresql status'" # checking postgres
```

File: Dockerfile [hunk 2]
```diff
@@ -75,5 +75,8 @@ COPY my_init.d/create-system-user.sh /etc/my_init.d/20_create-system-user.sh
 COPY opt/start-postgres.sh /opt/start-postgres.sh
 COPY my_init.d/start-postgres.sh /etc/my_init.d/30_start-postgres.sh
 
+# Health check
+HEALTHCHECK --interval=10s CMD su $SYSTEM_USER -c "/usr/lib/postgresql/10/bin/pg_ctl -D /home/$SYSTEM_USER/.postgresql status" || exit 1
```
Comment (ltalirz) [inline Dockerfile:79]: would be same here I guess
```suggestion
HEALTHCHECK --interval=10s CMD su $SYSTEM_USER -c "pg_ctl -D /home/$SYSTEM_USER/.postgresql status" || exit 1
```

**Review by ltalirz — COMMENTED** [review]
looks good, just one question

File: .travis.yml (hunk 1 above)
Comment (yakutovicha) [reply .travis.yml:29]: unfortunately not:
```
aiida@7f63eea5893e:/$ pg_ctl
bash: pg_ctl: command not found
```

**Comment by yakutovicha** [comment]
Of course, I can't call it a fix to the problem but at least it is clear where the path to PostgreSQL tools is provided.

**Review by ltalirz — APPROVED** [review]
ok, then let's go!

**Pull request merged**